### PR TITLE
add image build(s)

### PR DESCRIPTION
**Pull request by PeerHerholz**
- file generation through neurodocker
- docker and/or singularity
- option to build image(s) locally

### CODE REVIEW

**Comment by PeerHerholz** [comment]
@JoseAlanis, I just remembered that we need FreeSurfer for source reconstruction, etc., eh?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    && conda install -y -q --name mb-mm \
+           'python=3.6' \
+           'numpy' \
+           'pandas' \
+           'sklearn-lmer' \
+           'scipy' \
+           'rpy2=2.9.4' \
+           'r-lme4' \
+           'r-lmertest' \
+           'r-lsmeans' \
```
Comment (JoseAlanis) [inline Dockerfile]: lets go with ``emmeans`` instead of ``lsmeans``. I think their going to archive ``lsmeans`` in the near future.

**Comment by JoseAlanis** [comment]
> @JoseAlanis, I just remembered that we need FreeSurfer for source reconstruction, etc., eh?

well it depends I guess, do we wanna do source reconstruction here?

 I'll try to look for some other insteresting packages today (e.g., for viz).

**Review by JoseAlanis — COMMENTED** [review]
Do we wanna use ``ggplot2`` for vizualisation? We have some code for that alredy, right? What do you think, perhaps we could just reuse / imporve it?

**Comment by PeerHerholz** [comment]
> > @JoseAlanis, I just remembered that we need FreeSurfer for source reconstruction, etc., eh?
> 
> well it depends I guess, do we wanna do source reconstruction here?

Oh sorry, I totally mixed up the repos. I meant to post that in the [EEGprep docker PR](https://github.com/C0C0AN/EEGprep/pull/2). Never mind, we don't need FreeSurfer here.

**Comment by PeerHerholz** [comment]
> Do we wanna use `ggplot2` for vizualisation? We have some code for that alredy, right? What do you think, perhaps we could just reuse / imporve it?

Yes, this seems about right. The idea would be to generate informative `.html` files, that include basic statistics about the values (mindboggle output), that is an interactive version of the `.csv` file, distribution of the data, some tests (e.g., normality, etc.) and the outcomes of the mixed models as static and interactive graphic. The `.html` file will be based on a template and the add the respective information/graphics after the BIDSapp was run, with the corresponding files (e.g., summary `.csv`, model outcomes, etc.) saved as files themselves in the same folder. Do you have any idea if `ggplot2` works as `plotly` in that if embedded in a `.html` remains interactive even if the corresponding graphic is not shared along with the `.html`? It should if we directly include the produced `ggplot2` `.html` code, eh?

File: Dockerfile (hunk 1 above)
Comment (PeerHerholz) [reply Dockerfile]: kk, thx for the pointer. I will update this part.

Comment (PeerHerholz) [reply Dockerfile]: Done [here](https://github.com/C0C0AN/Mindboggle-MixedModels/pull/2/commits/e63eba433083f2566dae2b717b4c0a1ee201ef64).

**Pull request merged**